internal/venice-test-common/src/integrationtest/java/com/linkedin/venice/endToEnd/PartialUpdateTest.java

```diff
@@ -140,6 +142,139 @@ public void setUp() {
     this.parentController = parentControllers.get(0);
   }
 
+  @Test
+  public void testReproduceError() throws IOException {
```

Comment:
The test name is confusing.
Is this test mainly to validate the chunking support in repush?
If yes, can we rename it as something like `testRepushWithChunkingFlagChanged`?